digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/RoadAddressService.scala

```diff
@@ -651,11 +650,11 @@ class RoadAddressService(roadLinkService: RoadLinkService, eventbus: DigiroadEve
       rl.linkId -> buildRoadAddressLink(rl, ra, Seq())
     }.toMap
 
-    val (filledTopology, changeSet) = RoadAddressFiller.fillTopology(allRoadLinks, viiteRoadLinks)
+    val (filledTopology, changeSet) = RoadAddressFiller.fillTopology(roadLinksWithComplementary, viiteRoadLinks)
 
     publishChangeSet(changeSet)
 
-    filledTopology
+    filledTopology ++ suravageLinks
```

Comment:
If we don't use the suravage links on the fetch, they will not be returned anymore. Probably what we want is to don't have missing road address on top of suravage links.